### PR TITLE
Recorded Future V2 connector - Include threat map and threat indicators

### PR DESCRIPTION
**Pull request by lindarecfut**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/RecordedFutureV2/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,2617 +1,2466 @@
...
+x-ms-visibility: important
+items:
+$ref: '#/definitions/LinkEntities'
+LinkEntities:
+type: object
+properties:
+type:
+type: string
+title: type
+description: Enitity type
```
Comment (vmanoharas) [inline certified-connectors/RecordedFutureV2/apiDefinition.swagger.json]: Can you please correct the type "Enitity"

**Review by vmanoharas — COMMENTED** [review]
Hello @lindarecfut, 

The pipeline is experiencing internal issues, Could you please resubmit your PR/commit so that it can be triggered again and undergo validation? Thank you for your understanding. We apologize any inconvenience caused.

image

**Comment by vmanoharas** [comment]
Hello @lindarecfut, 

I am following up on above request, kindly resolve review comments. If this is not valid current, kindly close this PR.

**Comment by RecordedFutureOskbo** [comment]
> Hello @lindarecfut,
> 
> I am following up on above request, kindly resolve review comments. If this is not valid current, kindly close this PR.

Thanks! 
We will look into this and try fix. 

Mvh
Oskar

**Comment by vmanoharas** [comment]
Hello @lindarecfut, 

Our pipeline is failing. Could you please close this PR and submit new PR so that it can be triggered again and undergo validation? Thank you for your understanding. We apologize any inconvenience caused

**Comment by niklas-recordedfuture** [comment]
@vmanoharas I think the failing pipeline was due to us - `apiDefinition.swagger.json` contained non-JSON content.

We have now pushed a fix. Is it possible to re-run the checks and get a re-review?

But if you still want us to close the PR and re-open it we are happy to do it.

**Comment by vmanoharas** [comment]
Hello @lindarecfut, 

Unfortunately, the issue is not resolved yet, additionally license/CLA has not been passed.  Kindly close this PR and submit new PR.

**Comment by niklas-recordedfuture** [comment]
Hi @vmanoharas , I have opened a new PR here: https://github.com/microsoft/PowerPlatformConnectors/pull/3549

I don't have access to close this PR, could you please do it for me?

**Comment by vmanoharas** [comment]
Closing this PR as new PR created [3549](https://github.com/microsoft/PowerPlatformConnectors/pull/3549)

**Pull request closed without merging**